index.bs

```diff
@@ -823,16 +900,36 @@ The following example shows an [[#item-resource]] in the [[#ttl-representation]]
 
 ### Signed tree head ### {#signed-tree-head}
 
+The signed tree head for a register is available from the [[#register-proof-resource]], specifiying the proof-identifier as merkle:sha-256.
+
+The signed tree head for a register is the signed Merkle tree hash ([[RFC6962]] section 2.1) of a Merkle tree containing all entries in the register. 
+
 ISSUE(openregister/specification#29): What happens when a key is rotated?
 
 ### Leaf signature ### {#leaf-signature}
 
 ### Verifying the register ### {#ct-verify-register}
 
+To verify a register obtained from the [[#download]], a client must compute the Merkle tree hash from the complete list of raw [[#entry-resource]], as per [[RFC6962]] section 2.1, ensuring that it equals the tree-hash of the [[#register-proof-resource]] and that they can verify the signed tree head. 
```

Comment:
> and that they can verify the signed tree head

Would be clearer to explicitly talk about verifying a signature against a public key here.